Fruit Falling fixes

docEdub opened the pull request:
## Overview
Fixes issues I found when working through the "Fruit Falling" Ionic golden path at
https://doc.babylonjs.com/guidedLearning/devStories/fruitFalling.

## Details
I walked through the tutorial on a Mac mini M1 arm64 machine and hit several issues. Some were specific to Mac and arm64 and others affected all platforms. The tutorial implicitly assumes the Windows platform is being used, so this change only fixes the issues affecting all platforms. The Mac/arm64 issues I ran into can be addressed later if we decide to make it more cross-platform friendly.

See individual commit messages for details on each fix.

Temechon commented:
Thank you @brianzinn ! I'm going to add it in the index once it's merged.

deltakosh commented:
Hello! We just did a big update to the Doc. You may want to check it before to be sure your great content is where it Should:)

deltakosh commented:
@temechon just make sure to move the file to the right place:)

docEdub committed:
Update playground link to change `Mesh.Sphere` to `MeshBuilder.Sphere`

docEdub committed:
Change `player.jpg` to `player.png` to match linked playground

docEdub committed:
Fix raw git url in `10_raw_sprite_location.jpg`

This change updates the image to add the missing `public` part of the url.

docEdub committed:
Add `--access public` flag to `npm publish` command

When using a free npm account the `npm publish` command fails with:
ERR! 402 Payment Required - PUT https://registry.npmjs.org/@[...]fruit-fallin - You must sign up for private packages

There are two ways to avoid this error:
1. Add `"publishConfig": { "access": "public" }` to package.json.
2. Use the `--access public` command line argument.

I opted for the command line argument since users may not see it's set to public in package.json. Using the command line argument makes it obvious the package will be public.

docEdub committed:
Fix closing </canvas> tag

The Home.tsx example is missing the slash in it's `<canvas>` closing tag.

docEdub committed:
Fix command used to test Ionic app in browser

The `ionic serve` command should be used to test the Ionic app in the browser. This is listed correctly in an earlier section but in this section it's listed as `npm run serve`, which errors out with:
ERR! Missing script: "serve".

docEdub committed:
Call `initializeBabylonApp(...)` in Home.tsx

The Home.tsx example is missing the call to `initializeBabylonApp(babylonOptions)` after the options object is initialized. This makes the Ionic app show nothing but a white screen when the `ionic serve` command is run.

This change fixes the issue by adding the call to `initializeBabylonApp(babylonOptions)` immediately after the options object is initialized.

docEdub committed:
More player.png image filename and url fixes
